db/scripts/4_create_views.sql

```diff
@@ -56,7 +56,7 @@ ORDER BY l1.batch_index ASC
 CREATE OR REPLACE VIEW batchable_l2_only_tx_states
 AS
 
-SELECT tx.*, row_number() over (ORDER BY tx.id ASC) -1 AS row_number
+SELECT tx.*, row_number() over (ORDER BY tx.block_number ASC, tx.tx_index ASC) -1 AS row_number
```

Comment:
Ah so if you have multiple transactions in the same block then you order by the ID, that right? This keeps everything globally ordered

Reply:
I'm trying to replace all ordering on ID since that appears to be at least somewhat nondeterministic under load. If somehow we end up with 2 L2 txs in the same block (we shoudln't, that'd be an issue), we'll order _those_ by the tx index in the block.

Reply:
Ok great, that makes sense